static/app/components/sentryDocumentTitle.tsx

```diff
@@ -25,6 +25,10 @@ const DEFAULT_PAGE_TITLE = 'Sentry';
 
 const DocumentTitleContext = createContext(DEFAULT_PAGE_TITLE);
 
+export function useDocumentTitle() {
+  return useContext(DocumentTitleContext);
+}
```

Comment:
Exposing a hook so this value can be used automatically in the `BreadcrumbsFromContext` component